test_debugger.py::test_case_15 fails if parent directory name contains a "+"

I unzipped the latest PyDev.Debugger (https://github.com/fabioz/PyDev.Debugger/commit/1b1fb8b8a8d1c1068fecf2e28890a2e764eade71) into a directory called "PyDev.Debugger-main" and separately into a directory called "PyDev.Debugger+main" and discovered that the test `tests_python/test_debugger.py::test_case_15` fails in the second case, using both Python 3.9 and Python 3.10.  (I discovered this when trying to locate the source of some test failures in my attempts to create a Debian package for pydevd and then debugpy; still several more to go after this one!)

Here's the key part of the error messages, which appears as part of the Thread Dump MainThread:
```
E       Traceback (most recent call last):
E         File "/build/pydevd/PyDev.Debugger+main/_pydevd_bundle/pydevd_vars.py", line 196, in custom_operation
E           execfile(code_or_file, namespace, namespace)
E         File "/build/pydevd/PyDev.Debugger+main/_pydev_bundle/_pydev_execfile.py", line 14, in execfile
E           stream = tokenize.open(file)  # @UndefinedVariable
E         File "/usr/lib/python3.9/tokenize.py", line 392, in open
E           buffer = _builtin_open(filename, 'rb')
E       FileNotFoundError: [Errno 2] No such file or directory: '/build/pydevd/PyDev.Debugger main/tests_python/resources/_debugger_case15_execfile.py'
E       0.01s - sending cmd (quoted-line) -->            CMD_ERROR Error in running custom operationTraceback (most recent call last):   File "/build/pydevd/PyDev.Debugger+main/_pydevd_bundle/pydevd_comm.py", line 1667, in do_it     resEncoded = quote_plus(res)   File "/usr/lib/python3.9/urllib/parse.py", line 887, in [...]
```
I do not understand why the "+" has been converted into a space; somehow `tokenize.open(file)` has been given an incorrect filename.  This probably points to some error in PyDev.Debugger.

--- a/tests_python/debugger_unittest.py
+++ b/tests_python/debugger_unittest.py
@@ -1290,7 +1290,7 @@ class AbstractWriterThread(threading.Thread):
 
     def write_custom_operation(self, locator, style, codeOrFile, operation_fn_name):
         self.write("%s\t%s\t%s||%s\t%s\t%s" % (
-            CMD_RUN_CUSTOM_OPERATION, self.next_seq(), locator, style, codeOrFile, operation_fn_name))
+            CMD_RUN_CUSTOM_OPERATION, self.next_seq(), locator, style, quote_plus(codeOrFile), operation_fn_name))
 
     def write_evaluate_expression(self, locator, expression):
         self.write("%s\t%s\t%s\t%s\t1" % (CMD_EVALUATE_EXPRESSION, self.next_seq(), locator, expression))

